feat/build.gradle.kts

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation(project(":core"))
     implementation(project(":network"))
     implementation(project(":domain"))
+    implementation(project(":navigation"))
```

Comment:
Do not implement navigation for this. You need to implement the navigation service from the core module.